integreat_cms/cms/forms/translations/translations_management_form.py

```diff
@@ -1,17 +1,67 @@
 import logging
 
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
 from ...models import Region
 from ..custom_model_form import CustomModelForm
 
 
 logger = logging.getLogger(__name__)
 
 
+class CustomCheckboxSelectMultiple(forms.CheckboxSelectMultiple):
+    """
+    Custom CheckboxSelectMultiple child class which appends a machine
+    translation provider to option labels or disables them if no provider
+    is available
+    """
+
+    def create_option(self, name, value, *args, **kwargs):
+        r"""
+        This method extends the default ``create_option()``-method of the base
+        :class:`~django.forms.CheckboxSelectMultiple` to alter attributes set
+        during option creation.
+
+        :param name: unused arg containing the name of the option
+        :type name: str
+
+        :param value: The language tree node the option references
+        :type value: ~integreat_cms.cms.models.languages.language_tree_node.LanguageTreeNode
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The option dictionary
+        :rtype: dict
+        """
+
+        option = super().create_option(name, value, *args, **kwargs)
+        mt_provider = value.instance.mt_provider
+        if mt_provider:
+            option["label"] += f" ({mt_provider})"
+        else:
+            option["attrs"]["disabled"] = ""
+            option["attrs"]["checked"] = False
+            option["attrs"]["class"] = "bg-gray-100 border-gray-300"
+        return option
+
+
 class TranslationsManagementForm(CustomModelForm):
     """
     Form for modifying machine translation settings of a region
     """
 
+    machine_translatable_languages = forms.ModelMultipleChoiceField(
+        widget=CustomCheckboxSelectMultiple(),
+        queryset=None,
+        required=False,
```

Comment:
```suggestion
        required=False,
        label=_("Language Selection"),
```
To also translate the field name for error messages, I suggest to set an explicit label here.